lib/search.js

```diff
@@ -69,7 +69,9 @@ function search (args, silent, staleness, cb) {
     log.gauge.pulse('search')
     log.gauge.show({section: searchSection, logline: 'scanning ' + pkg.name})
     // Simply 'true' if the package matches search parameters.
-    var match = packageFilter(pkg, opts, searchexclude)
+    var match = packageFilter(pkg, opts, searchexclude, {
```

Comment:
Looking at this in the context of `filter()`, fixing this could be as simple as changing `opts` to `included`, and `searchexclude` to `excluded` to match.